funcx_endpoint/funcx_endpoint/endpoint/endpoint_manager.py

```diff
@@ -206,11 +204,29 @@ def start_endpoint(self, name, endpoint_uuid, endpoint_config):
             self.logger.exception("Caught exception while trying to setup endpoint context dirs")
             sys.exit(-1)
 
+        # Configure the parameters for the interchange
+        optionals = {}
+        if 'endpoint_address' in self.funcx_config:
+            optionals['interchange_address'] = self.funcx_config['endpoint_address']
+
+        optionals['logdir'] = endpoint_dir
+
+        if self.debug:
+            optionals['logging_level'] = logging.DEBUG
+
+        interchange = EndpointInterchange(endpoint_config.config,
```

Comment:
There seem to be issues with initializing this outside of the daemon context... I believe it is because of the interchange file logger. I think the file logger needs to be initialized in the daemon context for it to work